.github/workflows/automated-dev-tests.yml

```
@@ -602,145 +398,20 @@ jobs:
             !${{runner.workspace}}/openfast/build/reg_tests/glue-codes/openfast/UAE_VI
             !${{runner.workspace}}/openfast/build/reg_tests/glue-codes/openfast/WP_Baseline
 
-  rtest-openfast_io:
-    runs-on: ubuntu-22.04
-    needs: build-openfast-release
-    steps:
-      - name: Cache the workspace
-        uses: actions/cache@v4
-        with:
-          path: ${{runner.workspace}}
-          key: build-openfast-release-${{ github.sha }}
-      - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.11'
-      - name: Install dependencies
-        run: |
-          pip install -r requirements.txt
-          sudo apt-get update -y
-          sudo apt-get install -y libopenblas-dev libopenblas-openmp-dev
-          sudo apt-get install -y libhdf5-dev libnetcdf-dev libopenmpi-dev libyaml-cpp-dev
-      - name: Install openfast_io
-        working-directory: ${{runner.workspace}}/openfast/openfast_io
-        run: |
-          pip install -e .
-      - name: Configure Tests
-        working-directory: ${{runner.workspace}}/openfast/build
-        run: |
-          cmake \
-            -DPython_ROOT_DIR:STRING=${{env.pythonLocation}} \
-            -DBUILD_TESTING:BOOL=ON \
-            -DCTEST_PLOT_ERRORS:BOOL=ON \
-            ${GITHUB_WORKSPACE}
-          cmake --build . --target regression_test_controllers
-      - name: Run openfast_io tests
-        working-directory: ${{runner.workspace}}/openfast/build
-        run: |
-          ctest -VV -j4 \
-          -L openfast_io 
-      - name: Failing test artifacts
-        uses: actions/upload-artifact@v4
-        if: failure()
-        with:
-          name: rtest-openfast_io
-          path: |
-            ${{runner.workspace}}/openfast/build/reg_tests/openfast_io
 
-
-  rtest-OF-simulink:
-    runs-on: ubuntu-22.04
-    needs: build-openfast-release
+  rtest-OF-offshore:
+    runs-on: ubuntu-24.04
+    needs: build-all-release
```

```suggestion
    needs: build-all-release
    OMP_NUM_THREADS: 2
```
Setting to 2 threads here to match old method.  Might get the `5MW_OC4Jckt_DLL_WTurb_WavesIrr_MGrowth` to match the baseline (we don't have any artifacts from this case passing on `main` or `.log` files in the repo to check against).